Gemfile

```diff
@@ -54,6 +53,10 @@ gem 'tzinfo-data', platforms: %i(mingw mswin x64_mingw)
 gem 'uglifier'
 gem 'wdm', platforms: %i(mingw mswin x64_mingw)
 
+group :production do
```

Comment:
Ah, one thing is here is that I was keeping it in dev intentionally so we could use it at dev-time as well. Guess either way is fine though since most people wouldn't go get the APM token for their dev env.